test/v1/validation/AgentDetailsValidatorSpec.scala

```diff
@@ -48,8 +48,7 @@ class AgentDetailsValidatorSpec extends BaseSpec {
 
     "passed false and Some name (with incorrect name length) should not succeed with 2 errors" in {
       val model = AgentDetailsModel(false, Some(""))
-      leftSideError(model.validate).errorMessage shouldBe
-        errorMessages(AgentNameLengthError("").errorMessage, AgentNameSuppliedError("").errorMessage)
+      leftSideError(model.validate).errorMessage shouldBe AgentNameLengthError("").errorMessage
     }
```

Comment:
Here and other places where we were previously testing for two errors (or more) we can do this:
```leftSideError(model.validate, 0).errorMessage shouldBe AgentNameLengthError("").errorMessage
leftSideError(model.validate, 1).errorMessage shouldBe AgentNameSuppliedError("").errorMessage```

Giving the element to the `leftSideError` method lets us retain the testing for multiple errors